### DISCUSSION TITLE
Disable Botplay Text

### DISCUSSION THREAD

**Discussion by newintownreal** [KadeArchive/Kade-Engine#982] [Q&A] [answered]
Yes, The Title Says it All

**Comment by isakube** [comment, marked as answer]
alr nice one fam, i'll tell you how to do it anyway but prepare for the downfall of your youtube channel

In: PlayState.hx
Find: `add(botplayTxt)` (or something similar)

Remove that line or comment it out.

**Reply by newintownreal** [reply]
thx but I cant compile because I don't enough space in my pc

**Reply by isakube** [reply]
oof :(
